### ISSUE TITLE
Add option to update templates

### ISSUE THREAD

**Issue by shihanng** [shihanng/gig#25]

**Comment by seniorquico** [comment]
When are the templates updated? Or should we just remove the cache folder to force a re-download?

**Comment by shihanng** [comment]
> Or should we just remove the cache folder to force a re-download?

Yes, @seniorquico, you are right. Unfortunately this is "best" way to "update" for the moment :bow:.
